src/osd/osd_types.h

```diff
@@ -4777,6 +4777,13 @@ struct OSDOp {
    * @param out [out] combined data buffer
    */
   static void merge_osd_op_vector_out_data(vector<OSDOp>& ops, bufferlist& out);
```

Comment:
@tchaikov 
One of the reason I decide to use static function is seeing this function. The use case is very similar.